main.go

```diff
@@ -34,9 +30,9 @@ func checkForErrorsAndExit(err error) {
 	if err == nil {
 		os.Exit(0)
 	} else {
-		logger := util.CreateLogger("")
+		logger := util.CreateLogEntry("", "debug")
```

Comment:
Does this configure this logger to debug log level as its default? Would be nice for the error handling to support `--terragrunt-log-level` too...